packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -250,7 +253,7 @@ export class InsuredBridgeL1Client {
     // while the bot is running.
     const whitelistedTokenEvents = await this.bridgeAdmin.getPastEvents("WhitelistToken", blockSearchConfig);
     for (const whitelistedTokenEvent of whitelistedTokenEvents) {
-      const l1Token = whitelistedTokenEvent.returnValues.l1Token;
+      const l1Token = this.l1Web3.utils.toChecksumAddress(whitelistedTokenEvent.returnValues.l1Token);
```

Comment:
nit: please add `utils.toChecksumAddress` to `const { toBN, soliditySha3 } = Web3.utils;` at the top.